app/assets/stylesheets/application.css.scss

```diff
@@ -81,5 +82,61 @@ input {
 
 	img {
 		max-width: 100%;
+
 	}
-}
\ No newline at end of file
+}
+
+
+
+
+
+
+/* Navbar - Line starting here to end of document added by Sulai*/
+
+.navbar-default {
+  background-color: black;
+  border-color: #030033;
+}
+
+/* profile view */
+.profile_view { 
+  padding-top: 70px; 
+  width: 100%;
+  
+  .profile_info {
+  
+    }
```

Comment:
You should remove the `.profile_info` selector if you don't have any styles specific to it.